react-components/src/hooks/scenes/use3dScenes.tsx

```diff
@@ -398,6 +416,7 @@ function createGetScenesQuery(limit: number = 100) {
       sceneGroundPlanes: {
         sources: groundPlaneSourceWithProperties
       }
-    }
-  } as const satisfies Omit<QueryRequest, 'cursor' | 'parameters'>;
+    },
+    cursors: sceneCursor !== undefined ? { scenes: sceneCursor } : undefined
```

Comment:
Is this truncation of the cursor necessary here? Isn't the check in the `queryFunction` enough?